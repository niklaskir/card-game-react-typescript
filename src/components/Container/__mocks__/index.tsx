import React, { FC, ReactNode } from 'react';

const Index: FC<ReactNode> = ({ children }) => (
  <div id="container">{children}</div>
);

export default Index;
