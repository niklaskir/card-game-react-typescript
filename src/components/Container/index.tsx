import React, { FC, ReactNode } from 'react';

const Container: FC<ReactNode> = ({ children }) => (
  <div className="container mx-auto py-10 px-4">
    <div className="grid grid-cols-1 lg:grid-cols-3 gap-4">{children}</div>
  </div>
);

export default Container;
