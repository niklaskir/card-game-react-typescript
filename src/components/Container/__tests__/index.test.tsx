import React from 'react';
import renderer from 'react-test-renderer';
import Container from '../index';

describe('Container', function () {
  it('should render correctly', () => {
    const container = renderer
      .create(
        <Container>
          <div>Hello world</div>
        </Container>,
      )
      .toJSON();

    expect(container).toMatchSnapshot();
  });
});
