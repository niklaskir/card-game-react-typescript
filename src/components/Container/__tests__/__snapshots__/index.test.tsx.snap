// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render correctly 1`] = `
<div
  className="container mx-auto py-10 px-4"
>
  <div
    className="grid grid-cols-1 lg:grid-cols-3 gap-4"
  >
    <div>
      Hello world
    </div>
  </div>
</div>
`;
